### ISSUE TITLE
running "minimesos install" without parameters hangs

### ISSUE THREAD

**Issue by sadovnikov** [ContainerSolutions/minimesos#224]
This code will never return
```
        CommandInstall install = new CommandInstall();
        install.execute();
```

The same happens if use tries to get usage text of install command running `minimesos install`

Either `System.in` has to be tested on presents of data or a parameter `--in` has to be added to indicate JSON on standard input

**Comment by frankscholten** [comment]
Yes let's add a ``--stdin`` flag

**Comment by sadovnikov** [comment]
Done. PR is coming

```
vikmac:mini-mesos vik$ ./bin/minimesos install
Install a framework with Marathon
Usage: install [options]
  Options:
    --exposedHostPorts
       Expose the Mesos and Marathon UI ports on the host level (we recommend to
       enable this on Mac (e.g. when using docker-machine) and disable on Linux).
       Default: false
    --marathonFile
       Marathon JSON app install file location. Either this or --stdin parameter
       must be used
    --stdin
       Use JSON from standard import. Allow piping JSON from other processes.
       Either this or --marathonFile parameter must be used
       Default: false
```

**Comment by sadovnikov** [comment]
closed within #228
